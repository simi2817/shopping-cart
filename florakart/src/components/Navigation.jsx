import { Link } from 'react-router-dom';

const Navigation = () => {
    return (
        <div className="navigation">
            <nav>
                <Link className='navLink' to='/about'>About-Us</Link>
                <Link className='navLink' to='/categories'>Categories</Link>
                <Link className='navLink' to='/cart'>Cart</Link>
            </nav>
        </div>
    )
}

export default Navigation;