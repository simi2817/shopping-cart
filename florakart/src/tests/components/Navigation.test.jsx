import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { test, expect } from 'vitest';
import { MemoryRouter } from 'react-router-dom';
import Navigation from '../../components/Navigation.jsx';

test('renders a message', () => {
    const { container } = render(<MemoryRouter><Navigation /></MemoryRouter>);
    expect(container).toMatchSnapshot();
})