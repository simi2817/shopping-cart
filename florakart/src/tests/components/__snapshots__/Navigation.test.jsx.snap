// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders a message 1`] = `
<div>
  <div
    class="navigation"
  >
    <nav>
      <a
        class="navLink"
        href="/about"
      >
        About-Us
      </a>
      <a
        class="navLink"
        href="/categories"
      >
        Categories
      </a>
      <a
        class="navLink"
        href="/cart"
      >
        Cart
      </a>
    </nav>
  </div>
</div>
`;
